### PR TITLE
Add a note about unused registration flow

### PR DESCRIPTION
**Pull request by arianvp**

### CODE REVIEW

File: docs/reference/user/registration.md [hunk 1]
```diff
@@ -70,6 +70,8 @@ If the code is incorrect or if an incorrect code has been tried enough times, th
 
 ## Registration without pre-verification {#RefRegistrationNoPreverification}
 
+_NOTE: This flow is currently not used by any clients. At least this was the state on 2020-05-28_
```
Comment (fisx) [inline docs/reference/user/registration.md:73]: isn't it used by ephemeral users?

Comment (arianvp) [reply docs/reference/user/registration.md:73]: No; that's https://github.com/wireapp/wire-server/pull/1119/files#diff-b3cd391e15f6db83f9dd84f13d45a24bL108

Comment (jschaul) [reply docs/reference/user/registration.md:73]: Maybe document the "flow for ephemeral (wireless) users", too?

Comment (arianvp) [reply docs/reference/user/registration.md:73]: I'm confused. it's already documented no? (what  I linked to)

Comment (fisx) [reply docs/reference/user/registration.md:73]: Yes, it's dokumented.  I will merge this PR now, it's good.

**Pull request merged**